array_api_strict/__init__.py

```diff
@@ -1,117 +1,18 @@
-  is to restrict the namespace of the array object to only those dtypes and
-  only those methods that are required by the spec, as well as to limit/change
-  certain behavior that differs in the spec. In particular:
-
-  - The array API namespace does not have scalar objects, only 0-D arrays.
-    Operations on Array that would create a scalar in NumPy create a 0-D
-    array.
-
-  - Indexing: Only a subset of indices supported by NumPy are required by the
-    spec. The Array object restricts indexing to only allow those types of
-    indices that are required by the spec. See the docstring of the
-    array_api_strict.Array._validate_indices helper function for more
-    information.
-
-  - Type promotion: Some type promotion rules are different in the spec. In
-    particular, the spec does not have any value-based casting. The spec also
-    does not require cross-kind casting, like integer -> floating-point. Only
-    those promotions that are explicitly required by the array API
-    specification are allowed in this module. See NEP 47 for more info.
-
-  - Functions do not automatically call asarray() on their input, and will not
-    work if the input type is not Array. The exception is array creation
-    functions, and Python operators on the Array object, which accept Python
-    scalars of the same type as the array dtype.
-
-- All functions include type annotations, corresponding to those given in the
-  spec (see _typing.py for definitions of some custom types). These do not
-  currently fully pass mypy due to some limitations in mypy.
-
-- Dtype objects are just the NumPy dtype objects, e.g., float64 =
-  np.dtype('float64'). The spec does not require any behavior on these dtype
-  objects other than that they be accessible by name and be comparable by
-  equality, but it was considered too much extra complexity to create custom
-  objects to represent dtypes.
-
-- All places where the implementations in this submodule are known to deviate
-  from their corresponding functions in NumPy are marked with "# Note:"
-  comments.
-
-Still TODO in this module are:
-
-- DLPack support for numpy.ndarray is still in progress. See
-  https://github.com/numpy/numpy/pull/19083.
-
-- The copy=False keyword argument to asarray() is not yet implemented. This
-  requires support in numpy.asarray() first.
-
-- Some functions are not yet fully tested in the array API test suite, and may
-  require updates that are not yet known until the tests are written.
-
-- The spec is still in an RFC phase and may still have minor updates, which
-  will need to be reflected here.
-
-- Complex number support in array API spec is planned but not yet finalized,
-  as are the fft extension and certain linear algebra functions such as eig
-  that require complex dtypes.
+array_api_strict is a strict, minimal implementation of the Python array
+API (https://data-apis.org/array-api/latest/)
```

Comment:
this probably should be a markdown link around "Python array API"? and append "standard"?

Reply:
It's not a markdown link here because this is the docstring in a Python file.